README.md

```diff
@@ -1,9 +1,16 @@
+<p align="center"><img src="resources/logo.png" /></p>
+
 # @rootstrap/validate
 
 [![Maintainability](https://api.codeclimate.com/v1/badges/a878b4be647cd2a9582c/maintainability)](https://codeclimate.com/github/rootstrap/validate/maintainability) [![Test Coverage](https://api.codeclimate.com/v1/badges/a878b4be647cd2a9582c/test_coverage)](https://codeclimate.com/github/rootstrap/validate/test_coverage)
 
-[Description]
-Add the name and description of the library.
+Have you ever had to write your own custom validations? It can be a bit confusing, specially if you have never done it before. Don't worry, we are here to help.
+
+This library extends the popular library [validate.js](https://validatejs.org) and adds some custom validations that have proven to be very useful and quite commonly needed. Don't reinvent the wheel, next time you need to write a custom validation, check if someone hasn't implemented that custom validation already.
+
+Aside form common custom validations, this library will also serve as a gallery of all the custom validations that we have had to implement at Rootstrap, although some might not be included out of the box, we intend to showcase them in case you need them. [Here is a link]() to said gallery
```

Comment:
typo in form maybe?

Reply:
hehe, my achilles heel I swap those two words all the time.

Reply:
oh, I got it. It's validating a form. Nevermind

Reply:
but you are right! it should be `from` instead of `form`